src/main/java/com/shaft/support/DBActions.java

```diff
@@ -0,0 +1,155 @@
+package com.shaft.support;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.testng.Assert;
+
+import com.shaft.io.ReportManager;
+
+public class DBActions {
+
+	Connection connection;
+	Statement statement;
+	ResultSet resultSet;
+	String dbType;
```

Comment:
it is recommended that the dbType should be an enum with a list of the supported databases

Reply:
Done

Reply:
to-be-done in a future commit as agreed